shell/platform/darwin/macos/framework/Source/FlutterKeyHandler.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Cocoa/Cocoa.h>
+
+typedef void (^FlutterKeyHandlerCallback)(BOOL handled);
+
+/**
+ * An interface for a responder that can process a key event and decides whether
```

Comment:
Is this a handler, or a responder? If it's a responder, maybe call it `FlutterKeyResponder` instead?